cost-analyzer/values.yaml

```diff
@@ -520,3 +520,15 @@ serviceAccount:
 #  productKey: # apply business or enterprise product license
 #    key: ""
 #    enabled: false
+  # alertConfigs:
+  #   enabled: true
+  #   globalAlertEmails:
+  #   # - recipient@example.com
+  #   # - additionalRecipient@example.com
+  #   alerts:
+  #   # Daily budget alert on kubecost namespace at threshold 100 <currency>
+  #   - type: budget # supported: budget, weeklyUpdate
```

Comment:
can/should we generalize this to `recurringUpdate`? we will want to make timing configurable going forward

Reply:
I think we should add the `frequency` flag that you suggested last week, signifying how often to run the job. What do you think?

Reply:
i like the concept, but i propose we start with frequency == window

Reply:
agree with setting frequency and window to the same value for now.